u2flib_server/utils.py

```diff
@@ -56,8 +70,15 @@ def sha_256(data):
     return h.digest()
 
 
-Rand.rand_seed(os.urandom(1024))
+def rand_bytes(n_bytes):
+    return os.urandom(n_bytes)
 
 
-def rand_bytes(n_bytes):
-    return Rand.rand_bytes(n_bytes)
+def verify_ecdsa_signature(payload, pubkey, signature):
+    verifier = pubkey.verifier(signature, ec.ECDSA(hashes.SHA256()))
+    verifier.update(payload)
+
+    try:
+        verifier.verify()
+    except InvalidSignature:
+        raise Exception('Attestation signature verification failed!')
```

Comment:
Raising a generic `Exception` is not useful to users of this library.

I suggest either

1. Create a library specification hierarchy e.g. `U2fLibServerException` -> `VerifySignatureException`
2. Use a more specific stdlib exception e.g. `ValueError`
3. Allow the  `InvalidSignature` to pass, unmodified.

My vote would be for 1